server/lib/nl/common/utils.py

```diff
@@ -133,19 +149,20 @@ def _facet_contains_date(facet_data, facet_metadata, date) -> bool:
     obs_period_num = int(obs_period_num)
     # Difference in years between date of interest and earliest date
     # of the facet
-    year_diff = date.year - int(facet_earliest_date[:4])
-    if date.month:
+    year_diff = single_date.year - int(facet_earliest_date[:4])
+    if single_date.month:
       # Difference in number of months between the date of interest
       # and the earliest date of the facet
-      month_diff = year_diff * 12 + date.month - int(facet_earliest_date[5:])
+      month_diff = year_diff * 12 + single_date.month - int(
+          facet_earliest_date[5:])
```

Comment:
nit: I know this has been there before but would be great to add a comment about the magic number 5

Reply:
done

Reply:
Thanks!